src/java/org/apache/cassandra/config/CassandraRelevantEnv.java

```diff
@@ -61,6 +66,11 @@ public boolean getBoolean()
         return Boolean.parseBoolean(System.getenv(key));
     }
 
+    public boolean getBooleanOrDefault(Boolean defaultValue)
```

Comment:
nit:

```suggestion
    public boolean getBooleanOrDefault(boolean defaultValue)
```

this input allows `null` but will hit a NPE when returning; so good to block `Boolean` in favor of `boolean`